pkg/controller/pod.go

```diff
@@ -346,8 +360,9 @@ func (pnc *PodNetworksController) addNetworks(dynamicAttachmentRequest *DynamicA
 		return fmt.Errorf("failed to compute the updated network status: %v", err)
 	}
 
-	if err := nadutils.SetNetworkStatus(pnc.k8sClientSet, pod, newIfaceStatus); err != nil {
-		return err
+	err = annotations.SetNetworkStatus(pod, newIfaceStatus)
+	if err != nil {
```

Comment:
what's the purpose of this change ? 

Please keep the old format. It's OK to update the error msg.

Reply:
This updates the annotation in the pod object passed in parameter. The update of the pod via the Kubernetes API is no longer done here